obal/data/modules/srpm.py

```diff
@@ -0,0 +1,89 @@
+"""
+Build SRPM
+"""
+
+import shutil
+import os
+import commands
+from tempfile import mkdtemp
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.chdir import chdir
+
+def run_command(command):
+    return commands.getstatusoutput(command)
+
+def copy_sources(spec_file, package_dir, sources_dir):
+    command = "spectool --list-files %s" % spec_file
+    _status, sources = run_command(command)
+
+    source_files = []
+
+    for source in sources.split("\n"):
+        if 'http' not in source:
+            shutil.copy("%s/%s" % (package_dir, source.split(' ')[1]), sources_dir)
+        else:
+            with chdir(package_dir):
+                run_command("git-annex lock")
+                _status, annexed_files = run_command("git-annex find --include='*'")
+                annexed_files = annexed_files.splitlines()
+                run_command("git-annex get")
+                run_command("git-annex unlock")
+
+                for annex in annexed_files:
+                    shutil.copy(annex, sources_dir)
+
+                run_command("git-annex lock --force")
+
+
+def main():
+    """
+    Build a package using tito
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            package=dict(type='str', required=False),
+            output=dict(type='path', required=False),
+        )
+    )
+
+    package = module.params['package']
+    output = module.params['output']
+
+    spec_file = '%s/%s.spec' % (package, os.path.basename(package))
+
+    base_dir = mkdtemp()
+    sources_dir = os.path.join(base_dir, 'SOURCES')
+    build_dir = os.path.join(base_dir, 'BUILD')
+
+    os.mkdir(sources_dir)
+    os.mkdir(build_dir)
+
+    copy_sources(spec_file, package, sources_dir)
+    shutil.copy(spec_file, base_dir)
+
+    command = ['rpmbuild', '-bs']
+    command.append('--define "_topdir %s"' % base_dir)
+    command.append('--define "_sourcedir %s"' % sources_dir)
+    command.append('--define "_builddir %s"' % build_dir)
+    command.append('--define "_srcrpmdir %s"' % base_dir)
+    command.append('--define "_rpmdir %s"' % base_dir)
+    command.append('%s/%s.spec' % (base_dir, os.path.basename(package)))
```

Comment:
```suggestion
    command.append(spec_file)
```

Reply:
Note this is not the same path as the spec_file from up above. This is within the RPM builddirs.